twtr/grails-app/assets/javascripts/app/templates/home.tpl.htm

```diff
@@ -14,13 +14,14 @@
                    type="password" ng-model="auth.password" required />
         </div>
 
+        <div class="alert-danger" ng-show="errorMessage && errorMessage.length > 0">{{errorMessage}}</div><br/>
```

Comment:
not sure if this will populate with error message after the button click. Will give it a try once it is merged in